### ISSUE TITLE
Auth endpoint?

### ISSUE THREAD

**Issue by robn** [jmapio/jmap-perl#1]
Does the proxy have one yet? I don't think so, but that's now a problem because I need to discover the download endpoint to use a message blobId.

**Comment by neilj** [comment]
No, it doesn't at the moment. The download endpoint in the proxy is currently:

```
'/raw/{userId}/{blobId}/{name}'
```

where `userId` is the `k={userId}` from the URL.
